### DISCUSSION TITLE
Package libkf6windowsystem-dev not found in apt on Debian 12

### DISCUSSION THREAD

**Discussion by Flo1314** [tsujan/Kvantum#970] [Q&A]
Hey, I am on Debian 12 an would like to install libkf6windowsystem-dev for compilation.

For some reason it can not be found or installed via 

`sudo apt install libkf6windowsystem-dev`

Is there another way to install the package? :)

**Comment by tsujan** [comment]
On an LTS distro, you need to wait until its next version. By definition, an LTS distro doesn't provide you with recent apps/libraries.

BTW, currently `libkf6windowsystem-dev` doesn't exist in Debian Unstable either; it's only in Debian's Experimental repository.

**Reply by Flo1314** [reply]
Okay, thanks for the fast reply :)
